etc/cmake/options.cmake

```
@@ -141,6 +141,11 @@ if(OT_DNS_CLIENT)
     target_compile_definitions(ot-config INTERFACE "OPENTHREAD_CONFIG_DNS_CLIENT_ENABLE=1")
 endif()
 
+option(OT_DUCKHORN_BORDER_ROUTER "enable Duckhorn Border Router support")
```

```suggestion
option(OT_BORDER_ROUTING "enable Border Routing support")
```

Any suggestion on the name of the feature switch? @librasungirl

---

SGTM.